logstreamer/filehandling.go

```diff
@@ -471,7 +475,11 @@ func (ls *LogstreamSet) ScanForLogstreams() (result []string, errors *MultipleEr
 			if err != nil {
 				errors.AddMessage(err.Error())
 				position.Reset()
+				initialTail = ls.initialTail
+			} else if position.SeekPosition == 0 {
```

Comment:
This is pretty bad, the reason is though that LogstreamLocationFromFile will not return an error if journal is missing. Maybe there is a better way to do this?